src/components/PostDetails.tsx

```diff
@@ -1,118 +1,88 @@
 import React from 'react';
 import { Loader } from './Loader';
 import { NewCommentForm } from './NewCommentForm';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { CommentsList } from './CommentsList';
 
-export const PostDetails: React.FC = () => {
+type Props = {
+  activePost: Post,
+  postComments: Comment[] | null,
+  isCommentsError: boolean,
+  isNewCommentForm: boolean,
+  setIsNewCommentForm: React.Dispatch<React.SetStateAction<boolean>>,
+  postComment: (data: Omit<Comment, 'id'>) => Promise<void>,
+  isCommentLoading: boolean,
+  removeComment: (id: number) => Promise<void>
```

Comment:
one of the best naming strategies is name function in props as onAction, onChange, onRemoveComment ad so on.

`<Component onClick={handleClick} />`